src/braket/pennylane_plugin/translation.py

```diff
@@ -104,12 +105,18 @@ def supported_operations(device: Device) -> FrozenSet[str]:
         raise AttributeError("Device needs to have properties defined.")
     supported_ops = frozenset(op.lower() for op in properties.supportedOperations)
     supported_pragmas = frozenset(op.lower() for op in properties.supportedPragmas)
-    return frozenset(
+
+    op_set = frozenset(
         _BRAKET_TO_PENNYLANE_OPERATIONS[op]
         for op in _BRAKET_TO_PENNYLANE_OPERATIONS
         if op.lower() in supported_ops or f"braket_noise_{op.lower()}" in supported_pragmas
     )
 
+    if isinstance(device, AwsDevice) and device.arn == "arn:aws:braket:eu-west-2::device/qpu/oqc/Lucy":
```

Comment:
This is a mocking issue (since we can't connect to the hardware during testing, and have to use a mocked version). It looks like this is called in initialization, and the `._device._arn` attribute on the mocked version of the `AwsDevice` isn't added until _after_ the `AwsDevice` is initialized.

Reply:
Question: Are we adding pulse programming functionality onto the `braket.aws.qubit` device, are or we making a `braket.aws.pulse` device, that builds on the standard device but includes extra pulse-related functionality? 

If making a pulse device, we could just have this be a thing added in the device init instead of in the `supported_operations` function.

This also would tie into whether we want all braket devices to have a `settings` property, which would be `None` for most devices (and a dictionary for devices that you use with pulse programming), as well as things like whether we want to be including pulse validation checks in the standard braket device, etc. 

If we don't go with a separate category of device, maybe we should have a device attribute, something like `supports_pulses`, which we check for and set once in the init (before calling `supported_operations`). And then access this rather than the `device.arn` in this function. Because I think there are going to be quite a few places where we want to say "if this is pulse-based, do/check for these things, otherwise skip".

Reply:
I'm hesitant to add a `supports_pulses` attribute because as @Qottmann  pointed out, while Rigetti Aspen M3 also supports pulses, the braket plugin will not support that in this release, but having the `supports_pulse` attribute would imply that it does.

If we only set `supports_pulses` to true for the Lucy device, then I don't really see the point in adding that attribute since the device ARN is enough to determine pulse support, which we only need to figure out the supported gates anyway.